nibabel/freesurfer/tests/test_mghformat.py

```diff
@@ -345,11 +345,14 @@ def test_deprecated_fields():
 
     # mrparams is the only deprecated field at the moment
     # Accessing hdr_data is equivalent to accessing hdr, so double all checks
-    assert_array_equal(hdr['mrparams'], 0)
+    with pytest.deprecated_call():
```

Comment:
fyi. if you want to check specific warning, you can also use `with pytest.deprecated_call(match="deprecated from version: 2.3")`